AutomatedTesting/Gem/PythonTests/scripting/ScriptCanvas_TwoComponents_InteractSuccessfully.py

```diff
@@ -53,41 +37,56 @@ class TestScriptCanvas_TwoComponents_InteractSuccessfully:
 
     :return: None
     """
+    from editor_python_test_tools.utils import TestHelper
+    from editor_python_test_tools.utils import Report, Tracer
+    from editor_python_test_tools.editor_entity_utils import EditorEntity
+    from editor_python_test_tools.editor_component.editor_script_canvas import ScriptCanvasComponent
+    import scripting_utils.scripting_tools as scripting_tools
+    import azlmbr.legacy.general as general
+    import os
+    import azlmbr.paths as paths
+    import azlmbr.math as math
+    from scripting_utils.scripting_constants import (WAIT_TIME_3)
+
+    EXPECTED_LINES = ["Greetings from the first script", "Greetings from the second script"]
+    SOURCE_FILES = [os.path.join(paths.projectroot, "ScriptCanvas", "ScriptCanvas_TwoComponents0.scriptcanvas"),
```

Comment:
Let's break these out into two separate file consts to improve readability. I'm not seeing any reason why we should hide them in a list with only 2 entries.

Reply:
broke this up so that it's 2 files.I think it was  originally a list being processed by a helper file.